helpers/lib_tasks_docker_release.py

```diff
@@ -151,51 +383,20 @@ def docker_build_local_image(  # type: ignore
         ("POETRY_MODE", poetry_mode),
         ("CLEAN_UP_INSTALLATION", cleanup_installation),
     ]
-    build_args = " ".join("--build-arg %s=%s" % (k, v) for k, v in build_args)
+    build_args = " ".join(f"--build-arg {k}={v}" for k, v in build_args)
     # Build for both a single arch or multi-arch.
     if multi_arch:
         # Login to AWS ECR because for multi-arch we need to build the local
         # image remotely.
         hlitadoc.docker_login(ctx)
         # Create a multi-arch builder.
-        platform_builder = "multiarch_builder"
-        cmd = rf"""
-        docker buildx rm {platform_builder}
-        """
-        # We do not abort on error since the platform builder might be present
-        # or not from previous executions.
-        hsystem.system(cmd, abort_on_error=False)
-        cmd = rf"""
-        docker buildx create \
-            --name {platform_builder} \
-            --driver docker-container \
-            --bootstrap \
-            && \
-            docker buildx use {platform_builder}
-        """
-        hlitauti.run(ctx, cmd)
-        # Build.
-        # Compress the current directory (in order to dereference symbolic
-        # links) into a tar stream and pipes it to the `docker build` command.
-        # See HelpersTask197.
-        cmd = rf"""
-        tar -czh . | DOCKER_BUILDKIT={DOCKER_BUILDKIT} \
-            time \
-            docker buildx build \
-            {opts} \
-            --push \
-            --platform {multi_arch} \
-            {build_args} \
-            --tag {image_local} \
-            --file {dockerfile} \
-            -
-        """
-        hlitauti.run(ctx, cmd)
-        # Pull the image from registry.
-        cmd = rf"""
-        docker pull {image_local}
-        """
-        hlitauti.run(ctx, cmd)
+        _create_multiarch_builder(ctx)
+        # Build the multi-arch image.
```

Comment:
Looks like we don't need these comments anymore, as the function names are self-explanatory.